changes/2075.removal.8.rst

```diff
@@ -0,0 +1 @@
+The optional arguments of ``Command`` and ``Group`` are now keyword-only.
```

Comment:
Aside from being a generally good idea for functions with many optional arguments, this was necessary in order to make the `section` and `order` arguments appear in the same order between the two classes.